exercises/queen-attack/queen_attack.py

```diff
@@ -1,6 +1,6 @@
-def board():
+def board(pos1, pos2):
```

Comment:
What about `first_position` and `second_position`? It's more readable to me, don't you agree?